Packages/io.chainsafe.web3-unity/Runtime/Scripts/Connection/ReownConnectionProvider.cs

```diff
@@ -29,49 +33,34 @@ public class WalletConnectConnectionProvider : ConnectionProvider, IWalletConnec
 
         [field: SerializeField] public Metadata Metadata { get; private set; }
 
-        [field: SerializeField] public string StoragePath { get; private set; } = "wallet-connect/";
-
         [field: SerializeField] public string OverrideRegistryUri { get; private set; }
 
-        [DefaultAssetValue("Packages/io.chainsafe.web3-unity/Runtime/Prefabs/Wallet Connect/WalletConnect Dialog.prefab")]
-        [SerializeField] private ConnectionHandlerBehaviour handlerPrefab;
-
-        [SerializeField] private List<string> enabledWallets;
-
-        [SerializeField] private List<string> disabledWallets;
-
-        [field: SerializeField] public WalletConnectLogLevel LogLevel { get; private set; } = WalletConnectLogLevel.ErrorOnly;
+        [field: SerializeField] public ReownLogLevel LogLevel { get; private set; } = ReownLogLevel.ErrorOnly;
 
         [field: SerializeField]
         public WalletLocationOption WalletLocationOption { get; private set; } = WalletLocationOption.LocalAndRemote;
 
-        [field: SerializeField, DefaultAssetValue("Packages/io.chainsafe.web3-unity/Runtime/Sprites/Logo_WalletConnect.png")]
```

Comment:
Are you missing the Sprite of the Reown? If so maybe still reuse the WC one if there is no decet Reown logo.

Reply:
Yes, I'm missing one. Gonna add it now. But I actually removed the attribute cause I remembered that you can set default values for UnityEngine.Object child classes inside the inspector when you select a script. It also binds them by GUID and not the path, so you can move the asset freely.

Reply:
Added a new sprite for Reown.

Reply:
> you can set default values for UnityEngine.Object child classes inside the inspector when you select a script. It also binds them by GUID and not the path, so you can move the asset freely.

How do you do that?

Reply:
> How do you do that?

Just select the *.cs file in Unity, you will see it in the inspector.

Reply:
I had no idea xD, thanks!